solution/cmd.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+MODELS_DIR='/models'
+
+if [ -d "$MODELS_DIR" ]
+then
+	if [ ! "$(ls -A $MODELS_DIR)" ]; then
```

Comment:
Hey @ddnovikov, the startup script skipped downloading models and server failed to start -- my guess is this check. `/models` dir is not empty on start, it contains `lost+found` directory. Could you please modify the entrypoint script - say, just remove this `if` because we'll need to download the models anyway. Thanks

Reply:
here are full logs:
```bash
all done, starting the server
thread 'thread 'thread '<unnamed><unnamed><unnamed>' panicked at '' panicked at '' panicked at 'called `Result::unwrap()` on an `Err` value: TokenizerError("File not found error: /models/twitter-xlm-roberta-crypto-spam/sentencepiece.bpe.model vocabulary file not found :No such file or directory (os error 2)")called `Result::unwrap()` on an `Err` value: TokenizerError("File not found error: /models/Fake-News-Bert-Detect/vocab.json vocabulary file not found :No such file or directory (os error 2)")called `Result::unwrap()` on an `Err` value: TokenizerError("File not found error: /models/twitter-xlm-roberta-base-sentiment/sentencepiece.bpe.model vocabulary file not found :No such file or directory (os error 2)")', ', ', src/main.rssrc/main.rssrc/main.rs:::9393::935858:

58note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: TokenizerError("File not found error: /models/language-detection-fine-tuned-on-xlm-roberta-base/sentencepiece.bpe.model vocabulary file not found :No such file or directory (os error 2)")', src/main.rs:93thread ':<unnamed>58' panicked at '
called `Result::unwrap()` on an `Err` value: TokenizerError("File not found error: /models/xlm_roberta_base_multilingual_toxicity_classifier_plus/sentencepiece.bpe.model vocabulary file not found :No such file or directory (os error 2)")', src/main.rs:93:58
```